e2e/core/user-info-component-cloud.e2e.ts

```diff
@@ -15,20 +15,20 @@
  * limitations under the License.
  */
 
-import { LoginSSOPage } from '../pages/adf/loginSSOPage';
+import { LoginSSOPage } from '@alfresco/adf-testing';
 import { SettingsPage } from '../pages/adf/settingsPage';
 import TestConfig = require('../test.config');
 import { browser } from 'protractor';
 import { NavigationBarPage } from '../pages/adf/navigationBarPage';
-import { UserInfoDialog } from '../pages/adf/dialog/userInfoDialog';
+import { UserInfoDialogPage } from '@alfresco/adf-testing';
 import { Identity } from '../actions/APS-cloud/identity';
 
 describe('User Info - SSO', () => {
```

Comment:
This e2e file should be moved under the /e2e/process-cloud folder

Reply:
The userInfoDialog has got both Content and Process elements in it. Hence I have put it under core, for now.